packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -75,36 +72,53 @@ TIP: If your log file is part of your Salesforce DX project, you don’t need to
-  * **Copy as Expression** functions like Copy Value: It copies the variable’s value instead of copying the full variable name.
-  * **Add to Watch** copies the variable’s value into the WATCH section, but because we don’t evaluate variables in this section you see only `<VariableValue>:<VariableValue>`.
+- You can use this debugger only in your orgs. ISV customer debugging is unavailable in Apex Replay Debugger. To debug customers’ orgs, use [ISV Customer Debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger).
+- You can replay only one debug log at a time. This limitation can make it difficult to debug asynchronous Apex, which produces multiple debug logs.
+- Be sure to start a session soon after uploading your checkpoints, because checkpoints expire after 30 minutes.
+- Be sure to debug your code soon after starting the session, because heap dumps expire about a day after you generate them.
+- You can’t replay a debug log generated by scheduled Apex.
+- Long string variable values are truncated at breakpoints.
+- When viewing a standard or custom object, you can drill down only to its immediate child variables (one level deep).
+- You can’t expand a collection (a list, set, or map), because its members are shown in their string form.
+- Modifying a collection does not update the collection variable in the VARIABLES section of the Debug view.
+- You can’t set method or conditional breakpoints.
+- You can’t evaluate or watch variables or expressions in the Debug view’s WATCH section.
+- While debugging, right-clicking a variable in the VARIABLES section of the Debug view and selecting **Copy Value** works properly. However, **Copy as Expression** and **Add to Watch** don’t work as expected. 
+  - **Copy as Expression** functions like Copy Value: It copies the variable’s value instead of copying the full variable name.
+  - **Add to Watch** copies the variable’s value into the WATCH section, but because we don’t evaluate variables in this section you see only `<VariableValue>:<VariableValue>`.
 
 ## Bugs and Feedback
 To report issues with Salesforce Extensions for VS Code, open a [bug on GitHub](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Bug_report.md). If you would like to suggest a feature, create a [feature request on Github](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Feature_request.md).
 
 ## Resources
 
-* YouTube—Salesforce Releases: [Platform Services: Apex Replay Debugger](https://www.youtube.com/watch?v=8GVuMT4MHWc)
-* TrailheaDX ’18 session video: [Banish the Bugs: Apex Debuggers to the Rescue!](https://www.salesforce.com/video/2520334/)
+- YouTube—Salesforce Releases: [Platform Services: Apex Replay Debugger](https://www.youtube.com/watch?v=8GVuMT4MHWc)
```

Comment:
@lcampos We may want to get a video out there that shows replay debugging with checkpoints. That's not for this particular update but just a comment.